Docs/sphinx_documentation/source/AMReX_Profiling_Tools.rst

```diff
@@ -299,11 +320,18 @@ inside :cpp:`MyFunc0` and :cpp:`MyFunc1`:
               }
           }
 
-Currently, these are only available in the FullProfiler to control the size of the TinyProfiler output
-and allow instrumentation strategies without needing to re-code each time. You can use the scoped
-regions in a few select places that will be output in both profiler modes, while named regions
-will only be isolated during a FullProfiler run.
+Currently, these macros are only available in the Full Profiler. This was done to control the size of
+the Tiny Profiler output and to allow instrumentation strategies without needing to re-code each time.
+
 
+.. admonition:: A Combined Tiny-Full Region Profiling Strategy
+
+  We recommend using scoped regions in a few select places that will be timed when either the
+  Tiny or Full Profiler is activated. Because named regions will be isolated by the Full Profiler
+  and ignored by the Tiny Profiler, they can be used for regions that you wish to instrument only
+  when the Full Profiler is active.
+
+.. HACK verify intended meaning.
```

Comment:
I want to verify I understood this correctly.

Reply:
Not... quite. Regions don't add timers, just isolate sets of timers into a separate analysis. So, they do something like: "in this code block, these timers were found and took this long".

So, I would change line 329 to: "We recommend using scoped regions in a few select places that will be implemented when either the" (or something else that removed "timed", as regions don't add a timer.)

Reply:
Unfortunately, tiny profiler does add a timer to a region.

Reply:
Ah, yes. To get the "total time" of the region. There's isn't the extra REGION+TIMER variation, like in the Full Profiler.

I'd still lean away from using the word "timer", as we don't want people putting in REGIONS to act like "timers". Unless we want a full section discussing the output in substantial detail, this might be the way to go.